IPython/__init__.py

```diff
@@ -27,12 +27,12 @@
 #-----------------------------------------------------------------------------
 
 # Don't forget to also update setup.py when this changes!
-if sys.version_info < (3,3):
+if sys.version_info < (3,4):
     raise ImportError(
 """
-IPython 6.0+ does not support Python 2.6, 2.7, 3.0, 3.1, or 3.2.
+IPython 6.3+ supports Python 3.4 and above.
```

Comment:
There's some similar text to update in setup.py

https://github.com/takluyver/ipython/blob/68559994a2c1efcbd408fa63adaf1867389e22df/setup.py#L47

Reply:
Thanks, updated.